### DISCUSSION TITLE
l2tp cpu one core 100% occupied

### DISCUSSION THREAD

**Discussion by darkrain88** [SoftEtherVPN/SoftEtherVPN#2061] [Show and tell]
please help  to  fix the  bug  with  100% CPU  used  and  can't  connect with l2tp

thanks

**Comment by Evengard** [comment]
It is fixed, use the release https://github.com/SoftEtherVPN/SoftEtherVPN/releases/tag/5.02.5187

**Reply by darkrain88** [reply]
it is not merged in main branch?

**Reply by chipitsine** [reply]
it is there   https://github.com/SoftEtherVPN/SoftEtherVPN/commit/e3e0c33e3b48ee70d6334fb316c738876ab115ba


<img width="524" alt="image" src="https://github.com/user-attachments/assets/f25a4f67-b4ba-49d6-a170-60d969529dd4">

**Reply by chipitsine** [reply]
if you still observe the issue, please provide more details
